endpoint.go

```diff
@@ -833,6 +833,12 @@ func (ep *endpoint) Delete(force bool) error {
 				logrus.Warnf("failed to recreate endpoint in store %s : %v", name, e)
 			}
 		}
+
+		ep.releaseAddress()
```

Comment:
The current code makes sure that the resources are not freed up if deleteEndpoint fails.

I think the better fix is to make sandbox Delete() to call sb.delete(true), which will forcefully remove the endpoint.